### ISSUE TITLE
Notification - WCAG rule 1.3.1

### ISSUE THREAD

**Issue by SeamusLeonardHPE** [grommet/grommet#7602]
**Failed rule**

1.3.1 Info and Relationships

Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text.

---
**Notes**

The close button has an accessible name but it is not descriptive enough for a screenreader user to understand the relationship between the notification and the close button
